antsibull/build_ansible_commands.py

```diff
@@ -19,38 +19,56 @@
 import sh
 from jinja2 import Template
 from packaging.version import Version as PypiVer
+from semantic_version import Version as SemVer
 
 from . import app_context
 from .build_changelog import ReleaseNotes
 from .changelog import get_changelog
 from .collections import install_separately, install_together
 from .dependency_files import BuildFile, DependencyFileData, DepsFile
-from .galaxy import CollectionDownloader
+from .galaxy import CollectionDownloader, DownloadResults
 
 
 #
 # Common code
 #
 
 
-async def download_collections(deps, galaxy_url, download_dir, collection_cache=None):
+async def download_collections(deps: t.Dict[str, str],
```

Comment:
I read that input should be as loose as possible and output should be strict.  So I've been using `t.Mapping` for parameters (and `t.Dict` for return values if dict is what I'm returning).

Reply:
Fixed in b29791aa4070b0625f6845fe3382a7bca7dc17e0.